scipy/signal/filter_design.py

```diff
@@ -271,20 +272,22 @@ def freqz(b, a=1, worN=None, whole=False, plot=None):
         numerator of a linear filter
     a : array_like
         denominator of a linear filter
-    worN : {None, int, array_like}, optional
-        If None (default), then compute at 512 frequencies equally spaced
-        around the unit circle.
-        If a single integer, then compute at that many frequencies.
-        If an array_like, compute the response at the frequencies given (in
-        radians/sample).
+    worN : {int, array_like}, optional
+        If a single integer (default 512), then compute at that many
+        frequencies. Using a number that is fast for FFT computations
+        can result in faster computations (see Notes).
+        If an array_like, compute the response at the frequencies
+        given (in radians/sample).
     whole : bool, optional
         Normally, frequencies are computed from 0 to the Nyquist frequency,
-        pi radians/sample (upper-half of unit-circle).  If `whole` is True,
-        compute frequencies from 0 to 2*pi radians/sample.
+        π radians/sample (upper-half of unit-circle).  If True,
+        compute frequencies from 0 to 2π radians/sample.
```

Comment:
I think as long as the CIs come back happy it should be fine, and [CircleCI looks good](https://3836-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.signal.freqz.html#scipy.signal.freqz) so hopefully we're okay.